msg/BeaconRange.msg

```diff
@@ -0,0 +1,5 @@
+uint64 timestamp	    # time since system start (microseconds)
```

Comment:
Just curious, is the assumption that timestamps between the vehicle and the beacon synchronized? In case you have multiple beacons, how is this done?